app/konnect/configure/runtime-manager/index.md

```diff
@@ -26,33 +32,44 @@ The Runtime Manager aims to simplify this process by providing a
 script to provision a {{site.base_gateway}} runtime in a Docker container,
 eliminating any confusion about initial configuration or setup.
 
-## Types of runtimes
+## Runtime groups
 
-### Kong Gateway
+Konnect manages runtime configuration in runtime groups. Each runtime group acts
+as a separate control plane and can manage runtime configurations independently
+of any other group. See [runtime groups](/konnect/configure/runtime-manager/runtime-groups)
+for more information.
 
-A {{site.base_gateway}} runtime acts as a data plane, which is a node
-serving traffic for the proxy. Data plane nodes are not directly connected
-to a database.
+## Types of runtimes
+
+Every runtime instance in a group must be of the same type.
```

Comment:
I'd remove this + the following explanation until we support more than just Gateway. I wasn't thinking about multiple runtime types until I read it here